crates/rust-fnv/diff.txt

```diff
@@ -0,0 +1,7425 @@
+diff -Naur '--exclude=.git' crates/rust-fnv/.drone.yml ../rust-fnv/.drone.yml
```

Comment:
No need to put a diff .txt here. Copy it to the PR to make reviewing/auditing easier.

Also, we only need diff of changes on the original project. Please exclude additional  the sgx-tests directory.